### PR TITLE
Add SEASR 2016

### PR DESCRIPTION
**Pull request by jolene-esposito**
Add SEASR 2016 and update NCIII 'name' field

### CODE REVIEW

File: scripts/populate_conferences.py [hunk 1]
```diff
@@ -350,7 +350,15 @@ def main():
         'public_projects': True,
     },
 }
```
Comment (sloria) [inline scripts/populate_conferences.py]: Need a comma here because this is a list.

File: scripts/populate_conferences.py [hunk 2]
```diff
@@ -350,7 +350,15 @@ def main():
...
-
+    'SEASR2016': {
```
Comment (sloria) [inline scripts/populate_conferences.py]: Indentation is significant in Python, so this needs to be dedented.

**Comment by sloria** [comment]
Pass finished. :point_up:

**Comment by jolene-esposito** [comment]
I submitted a new PR with regards to these changes.

**Comment by HarryRybacki** [comment]
[#OSF-4272](https://openscience.atlassian.net/browse/OSF-4272)

**Pull request closed without merging**